all.sln

```diff
@@ -52,6 +52,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		.editorconfig = .editorconfig
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Extensions.Configuration.DaprSecretStore", "src\Dapr.SecretStore.ConfigurationExtension\Microsoft.Extensions.Configuration.DaprSecretStore.csproj", "{3EFB6020-F928-4FF4-A795-6B426C3886A9}"
```

Comment:
Change the project name and nuget to start with Dapr.

Reply:
Solved in ea0bc1b